### PR TITLE
feat(sidecar): consensus state validation

### PR DESCRIPTION
**Pull request by namn-grg**

### CODE REVIEW

File: bolt-sidecar/src/state/consensus.rs [hunk 1]
```diff
@@ -40,6 +84,37 @@ impl ConsensusState {
...
+        Ok(())
+    }
+}
+
+/// Get the current timestamp.
+fn current_timestamp() -> u64 {
+    return SystemTime::now()
+        .duration_since(UNIX_EPOCH)
+        .unwrap()
+        .as_secs();
```
Comment (namn-grg) [inline bolt-sidecar/src/state/consensus.rs:119]: Just a util function, ideally should be in utils.rs (not sure of the file structure in rust)

Comment (namn-grg) [reply bolt-sidecar/src/state/consensus.rs:119]: NOTE - the precision of the deadline is in seconds. We might want to do it in mili seconds or smaller

Comment (merklefruit) [reply bolt-sidecar/src/state/consensus.rs:119]: Seconds is fine for now!

**Review by merklefruit — APPROVED** [review]
This is in the right direction! Happy to merge as is and iterate over it in further PRs to unlock some other work on this part.

**Pull request merged**